src/api/.keystone/config.js

```diff
@@ -33,9 +33,33 @@ __export(keystone_exports, {
   default: () => keystone_default
 });
 module.exports = __toCommonJS(keystone_exports);
-var import_config2 = require("dotenv/config");
+var import_config3 = require("dotenv/config");
 var import_core2 = require("@keystone-6/core");
 
+// middleware/headers/check-api-key/index.ts
+var import_config = require("dotenv/config");
+var { API_KEY } = process.env;
+var checkApiKey = (req, res, next) => {
+  const { "x-api-key": xApiKey } = req.headers;
+  if (xApiKey !== API_KEY) {
+    return res.status(401).send("Unauthorised");
+  }
+  next();
+};
+var check_api_key_default = checkApiKey;
+
+// middleware/rate-limiter/index.js
+var import_express_rate_limit = __toESM(require("express-rate-limit"));
```

Comment:
out of curiosity as I'm not familiar with the package how does this work? Is it 1000 requests a minute per express user? or across the whole API? (And does this mean that it's per server hosted in azure that we have this rate limit?)

Reply:
That's is, this is across the whole API so after 1000 requests in a minute the rate limiter will reject calls. We have the exact same in place for the UI and probably in some DTFS services.

AFAIK there's nothing specific for rate limiting in Azure setup as this handles it